### DISCUSSION TITLE
the-risks-of-learning-a-sharepoint-alternative/

### DISCUSSION THREAD

**Discussion by martijnboland** [martijnboland/blog#37] [Blog comments]
# the-risks-of-learning-a-sharepoint-alternative/

https://blogs.taiga.nl/martijn/posts/the-risks-of-learning-a-sharepoint-alternative/

**Comment by disqus-comments-migrator** [comment]
<em>Nico Jansen commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at februari 07 2010, 02:49</em>

---

<p>Amen .... In dutch we would say .... je slaat de spijker op zijn kop ...</p>
